code/modules/mob/living/living_movement.dm

```diff
@@ -50,7 +50,7 @@
 	remove_movespeed_modifier(MOVESPEED_ID_PRONE_DRAGGING)
 
 /mob/living/can_zFall(turf/T, levels)
-	return !(movement_type & FLYING)
+	return ..() //!(movement_type & FLYING) 

```

Comment:
why comment out the old code?

Reply:
atom/can_zFall hold all what need to make mob dont fall.
But it still need to be upgrade to hold mechanics from another dont standard sources like mutations, equipment, etc.
Just remove commented part ?